### PR TITLE
Bookings show

### LINKED ISSUE #34: Get booking by yard ID
GET /api/v1/yards/yard_id/bookings

### PR DESCRIPTION
**Pull request by domo2192**
added yards/:id/bookings route 
index method for Bookings controller inside yards folder

Testing 
multiple sad path and happy paths

### CODE REVIEW

File: app/controllers/api/v1/yards/bookings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+module Api
+  module V1
+    module Yards
+      class BookingsController < ApplicationController
```
Comment (jbranham1) [inline app/controllers/api/v1/yards/bookings_controller.rb]: Could we be consistent with the rest of our app and do the syntax of `Api::v1::Yards::BookingsController`?

**Review by jbranham1 — CHANGES_REQUESTED** [review]
Overall looks great! I left one comment

File: app/controllers/api/v1/yards/bookings_controller.rb (hunk 1 above)
Comment (domo2192) [reply app/controllers/api/v1/yards/bookings_controller.rb]: Ya definitely I'll change that. If we are going to use rubocop at some point its going to change it to this way but we can learn how to set some ignore on it

**Review by Gvieve — APPROVED** [review]
This looks awesome! Great job

**Review by jbranham1 — APPROVED** [review]

**Pull request merged**